_1327/documents/views.py

```diff
@@ -44,7 +44,7 @@ def create(request, document_type):
 		model_class = content_type.model_class()
 		delete_old_empty_pages()
 		title = model_class.generate_new_title()
-		url_title = model_class.generate_default_slug(title)
+		url_title = "temp_{}_{}".format(datetime.utcnow().strftime("%d%m%Y%H%M%S%f"), model_class.generate_default_slug(title))
```

Comment:
`max_length` is 255 for `url_title`. this lowers the available number of characters quite a bit. increasing the length might not be necessary, but the forms need to make sure, that the given url title can actually be saved and that the user does not enter a 250 character long title there.

Reply:
what do you want me to do?
Add constraints to the forms?
Shorten the size of the timestamp?

Reply:
adding a constraint to the form sounds good. just throw an error in cleaning if the value is too long.

Reply:
I just saw, that the form already contains such a constraint :smile: